### ISSUE TITLE
Test new drawer toggle pattern for a11y

### ISSUE THREAD

**Issue by kimisgold** [omeka/plugin-Editorial#66]
We're revising Omeka Classic's collapsible drawer pattern and applying it to the plugins. This pattern includes components that can collapse and expand, as well as marking them for deletion. In Editorial, it appears in its block form on the exhibit page edit interface.

Test the following navigating via keyboard while also using screen readers for mobile and desktop browsers:

* The layout options drawer expands and collapses with the expected screen reader announcements.

**Comment by aviannamiller** [comment]
The toggle itself is functional and the announcement is perceived on all tested screen readers (NVDA, JAWS, VoiceOver, TalkBack). However, I am experiencing the following issues:
- It appears the expand/collapse pattern has been applied in reverse, in which the expanded toggle has a readout of collapsed and vice versa. I see `aria-expanded="true"` applied when the toggle is collapsed, and the reverse when it is expanded.
- I'm experiencing an odd bug when navigating with JAWS. When focus is brought to the toggle, focus immediately jumps to the page's h1. I can't see a way to activate the toggle via keyboard without this occurring (except for navigating in reverse? shift+tabbing back to the toggle oddly allows you to remain focus on it).

**Comment by kimisgold** [comment]
The `aria-expanded` value should be fixed now.

With regards to the JAWS bug, I suspect this is an ongoing issue with the screen reader. [I've filed a similar bug that happens with tinymce, where some focus switch scrolls the viewport back to the top of the page.](https://github.com/tinymce/tinymce/issues/8173#issuecomment-2831684653) I've seen other issues filed on the JAWS repo with similar instances of jumping to the top of the page after some kind of focus change. (Examples: https://github.com/FreedomScientific/standards-support/issues/797, https://github.com/FreedomScientific/standards-support/issues/774) I'm tempted to chalk this up to something that needs to be solved on JAWS' end.

**Comment by kimisgold** [comment]
I've added aria labelling for the layout options drawer group, similar to treatment for omeka/plugin-ExhibitImageAnnotation#14

**Comment by aviannamiller** [comment]
Expand/collapse pattern and updating labelling look great, so that should essentially be it for this issue.

Of course, there is that JAWS bug. I retested, and it does not appear to be browser-specific by any means. The example you linked does indeed echo this situation, so certainly something to look out for down the road!
